artiq/frontend/artiq_client.py

```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env python3
+"""
+Client to send commands to :mod:`artiq_master` and display results locally.
+
+The client can perform actions such as accessing/setting data, scanning devices,
```

Comment:
"datasets" to use consistent terminology.

Reply:
Fixed in https://github.com/m-labs/artiq/pull/1185/commits/c3206d79c7b535bc7c48b5b19844a10077c9ee49